test/test_shape_ops.py

```diff
@@ -406,6 +406,19 @@ def gen_data():
             out_t = make_from_data([[3, 2, 1], [6, 5, 4]])
             yield in_t, dims, out_t
 
+            # N3HW case (images)
+            if device == "cpu" and dtype != torch.bfloat16:
+                in_t = make_from_size((2, 3, 8, 9))
```

Comment:
Can you include some other channel sizes and also some non-contiguous cases to be sure the vectorized path isn't taken.